Source/Umbraco/JS/FormEditor/FormEditorAsync.js

```diff
@@ -275,7 +275,7 @@ addFormEditorCondition("core.fieldisempty", function (rule, fieldValue, formData
 });
 // - "field value is not" condition:
 addFormEditorCondition("core.fieldvalueisnot", function (rule, fieldValue, formData) {
-  return (fieldValue || "").toLowerCase() != (rule.condition.expectedFieldValue || "").toLowerCase();
+  return (fieldValue + "" || "").toLowerCase() != (rule.condition.expectedFieldValue || "").toLowerCase();
```

Comment:
`rule.condition.expectedFieldValue` will always be a string. But i guess you can't be too safe ;)